processor/priorityqueue.go

```diff
@@ -2,73 +2,9 @@ package processor
 
```

Comment:
After those changes, does the filename still remains relevant?

Reply:
We can rename it to transformQueue.go?

Reply:
all lowercase to be safe